Sources/SwiftToolchain/Toolchain.swift

```diff
@@ -283,21 +283,21 @@ public final class Toolchain {
     return testBundlePath
   }
 
-  public func packageInit(name: String, type: PackageType, inPlace: Bool) throws {
-    var initArgs = [
+  public func runPackageInit(name: String, type: PackageType, initialiseInSubdirectory: Bool) throws {
```

Comment:
I'm all for British spelling dominating the world, but consistency wins after all. Let's use the US one in code for consistency with other frameworks that have `Color` and other heretical spelling variants in them 😄

```suggestion
  public func runPackageInit(name: String, type: PackageType, initializeInSubdirectory: Bool) throws {
```

Reply:
Ok. I'll change the spellings. I can't commit the suggested change directly, though. Because I need to change the call sites as well.